docs/4.3/installation.md

```diff
@@ -89,53 +90,49 @@ $ helm install teleport gravitational/teleport
 
 ## MacOS
 
-**Package**
+=== "Homebrew"
 
-* **Option 1:** Download the `.pkg` file via your web browser and double-click
-  to run the Installer.
-* **Option 2:** Install on the command line with `installer`
-```bash
-$ curl -O https://get.gravitational.com/teleport-{{ teleport.version }}.pkg
-$ sudo installer -pkg teleport-{{ teleport.version }}.pkg -target / # Installs on Macintosh HD
-Password:
-installer: Package name is teleport-{{ teleport.version }}
-installer: Upgrading at base path /
-installer: The upgrade was successful.
-$ which teleport
-/usr/local/bin/teleport
-```
+    ```bash
+    $ brew install teleport
+    ```
+
+=== "Download"
+
+      [Download MacOS .pkg installer](https://get.gravitational.com/tsh-{{ teleport.version }}.pkg) (tsh client only, signed) file, double-click to run the Installer.
+
+=== "Terminal"
+
+    ```bash
+    $ curl -O https://get.gravitational.com/teleport-{{ teleport.version }}.pkg
+    $ sudo installer -pkg teleport-{{ teleport.version }}.pkg -target / # Installs on Macintosh HD
+    Password:
+    installer: Package name is teleport-{{ teleport.version }}
+    installer: Upgrading at base path /
+    installer: The upgrade was successful.
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
 
-**Tarball**
-```bash
-$ curl https://get.gravitational.com/teleport-v{{ teleport.version }}-darwin-amd64-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-darwin-amd64-bin.tar.gz
-$ shasum -a 256 teleport-v{{ teleport.version }}-darwin-amd64-bin.tar.gz
-# Verify that the checksums match
-$ tar -xzf teleport-v{{ teleport.version }}-darwin-amd64-bin.tar.gz
-$ cd teleport
-$ ./install
-$ which teleport
-/usr/local/bin/teleport
-```
 
 ## Windows (tsh client only)
 
 As of version v3.0.1 we have `tsh` client binary available for Windows 64-bit
 architecture - `teleport` and `tctl` are not supported.
 
-```bash
-> curl https://get.gravitational.com/teleport-v{{ teleport.version }}-windows-amd64-bin.zip.sha256
-# <checksum> <filename>
-> curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-windows-amd64-bin.zip
-> echo %PATH% # Edit %PATH% if necessary
-> certUtil -hashfile teleport-v{{ teleport.version }}-windows-amd64-bin.zip SHA256
-SHA256 hash of teleport-v{{ teleport.version }}-windows-amd64-bin.zip:
-# <checksum> <filename>
-CertUtil: -hashfile command completed successfully.
-# Verify that the checksums match
-# Move `tsh` to your %PATH%
-```
+=== "Poweshell"
+
+    ```bash
+    > curl https://get.gravitational.com/teleport-v{{ teleport.version }}-windows-amd64-bin.zip.sha256
+    # <checksum> <filename>
+    > curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-windows-amd64-bin.zip
```

Comment:
```suggestion
    > curl -O teleport-v{{ teleport.version }}-windows-amd64-bin.zip https://get.gravitational.com/teleport-v{{ teleport.version }}-windows-amd64-bin.zip
```

(you have to explicitly provide an output path otherwise it doesn't save the file)